src/src/winrm_bootstrap.txt

```diff
@@ -0,0 +1,37 @@
+<powershell>
```

Comment:
Why does this duplicate file exist?

Reply:
Sorry, need to do some clean up with the work done on this branch

Reply:
The duplicate file was removed in commit e2addb8.